python/ray/autoscaler/_private/util.py

```diff
@@ -144,6 +145,13 @@ def merge_setup_commands(config):
     return config
 
 
+def fill_node_type_max_workers(config):
+    """Default per-node max workers to global max_workers."""
+    assert "max_workers" in config, "Global max workers should be set."
+    for node_type in config["available_node_types"].values():
+        node_type["max_workers"] = config["max_workers"]
```

Comment:
this will overwrite the user's input!
you can do something like:
```suggestion
        node_type["max_workers"] = node_type.get("max_workers", config["max_workers"])
```

Reply:
oops, fixed